python/tests/unit/arcticdb/test_import.py

```diff
@@ -6,6 +6,36 @@
 As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
 """
 
+import arcticdb as adb
+
 
 def test_can_import_native_library():
     import arcticdb_ext
+
+
+def test_top_level_imports():
+    imports_list = [
+        "Arctic",
+        "LibraryOptions",
+        "QueryBuilder",
+        "QueryBuilder",
+        "VersionedItem",
+        "library",
+        "LibraryOptions",
+        "set_config_from_env_vars",
+        "DataError",
+        "VersionRequestType",
+        "ErrorCode",
+        "ErrorCategory",
+        "WritePayload",
+        "ReadInfoRequest",
+        "ReadRequest",
+        "StagedDataFinalizeMethod",
+        "WriteMetadataPayload",
+    ]
+    find_imports(imports_list)
+
+
+def find_imports(list_imports):
```

Comment:
I thought it would be better because `find_imports` is a general functionality. For example I could make this function slightly better like `def find_imports(library, imports_list)` and then you can test that for any given library, not just top-level-imports. But I can change it to your suggestion if you don't think it's helpful. Let me know your thoughts.